cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/hbase/HBaseQueueClientFactory.java

```diff
@@ -109,7 +114,7 @@ private HBaseQueueAdmin ensureTableExists(QueueName queueName) throws IOExceptio
   }
 
   private HTable createHTable(String name) throws IOException {
```

Comment:
Should get rid of this method and call `HBaseQueueAdmin` to do it (by taking a `QueueName` rather than a `String`).